Yoast/wordpress-seo
# Fix custom fields with spaces are erroneously duplicated on reload

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Needed to append a fix to a branch that was merged already. The issue was that custom fields and taxonomies with spaces in them were added to redux once with a space, and once with an underscore (on page refresh). The labels were similar however, so an array with only the labels (such as for the suggestions) had duplicate entries.

## Test instructions

This PR can be tested by following these steps:

* Before checking out this branch, create a custom field with a space in it.
* Check the `SnippetEditor>ReplacementVariables` field in the redux store. It will have your variable only once, with an underscore.
* Now reload the page, it will have the variable twice, once with, and once without an underscore. Opening the suggestions menu will throw warnings (when using the related yoast-components branch).
* Checkout branch and see that it works now.

Needs https://github.com/Yoast/yoast-components/issues/577
